woocommerce/woocommerce
# Add help tip for Product Image and Product Gallery meta boxes

### All Submissions:






### Changes proposed in this Pull Request:

Adds a tooltip for both "Set product image" and "Add product gallery images" buttons

<img width="304" alt="image" src="https://user-images.githubusercontent.com/13437655/205759951-6d434ed0-28eb-40b5-a748-7fbcd324e358.png">

<img width="306" alt="image" src="https://user-images.githubusercontent.com/13437655/205760111-2b2af113-7a63-44a1-b9e9-195526b26106.png">





Closes #35017.






### How to test the changes in this Pull Request:

1. Open Add/Edit product page.
2. Verify the updated UI of the Product Image and Product Gallery meta boxes






### Other information:




### FOR PR REVIEWER ONLY:
